lib/datasource/orb/__snapshots__/index.spec.ts.snap

```diff
@@ -153,7 +153,7 @@ Array [
     "headers": Object {
       "accept": "application/json",
       "accept-encoding": "gzip, deflate",
-      "content-length": 128,
+      "content-length": "128",
```

Comment:
?

Reply:
It comes from the options normalization done by got v11

Reply:
yes, ist the new way got will normalize all http headers.